### PR TITLE
LoginPage: Fix hover color for service buttons

### LINKED ISSUE #25007: LoginPage: White theme leaves some button text black & button black (for generic oauth & SAML)
image

### PR DESCRIPTION
**Pull request by tskarhed**
Fixes #25007

### CODE REVIEW

File: public/app/core/components/Login/LoginServiceButtons.tsx [hunk 1]
```diff
@@ -128,7 +131,7 @@ export const LoginServiceButtons = () => {
     return (
       <a
         key={key}
-        className={`${styles.button} btn btn-medium btn-service btn-service--${service.className || key}`}
+        className={`btn btn-medium btn-service btn-service--${service.className || key} ${styles.button}`}
```
Comment (Clarity-89) [inline public/app/core/components/Login/LoginServiceButtons.tsx:134]: Should we use `cx` here to concat class names?

**Review by Clarity-89 — APPROVED** [review]
Looks good, left a minor remark. So does this fix the enterprise issue as well? 

I think you need to update to latest master to fix the build issue.

File: public/app/core/components/Login/LoginServiceButtons.tsx (hunk 1 above)
Comment (tskarhed) [reply public/app/core/components/Login/LoginServiceButtons.tsx:134]: I think keeping the classnames until they are migrated to Emotion makes sense.

Comment (Clarity-89) [reply public/app/core/components/Login/LoginServiceButtons.tsx:134]: What I mean is can't you do smth like 
```
className={cx(`btn btn-medium btn-service btn-service--${service.className || key}`, styles.button}`)}
```

Comment (tskarhed) [reply public/app/core/components/Login/LoginServiceButtons.tsx:134]: Ah, fair enough

**Pull request merged**